Session/Conversations/ConversationVC.swift

```diff
@@ -215,6 +216,23 @@ final class ConversationVC: BaseVC, LibSessionRespondingViewController, Conversa
         return result
     }()
     
+    lazy var versionSupporBanner: InfoBanner = {
```

Comment:
So this banner is actually on the wrong screen (it should be on `HomeVC` rather than `ConversationVC`) - the naming we use internally is a little messy but generally:
- `HomeVC`: "Home Screen" or "Conversation List"
- `ConversationVC`: "Conversation Screen" or "Message List" (the latter is rarely used)

When adding it to the `HomeVC` please add a `showVersionSupportBanner` flag to the `HomeViewModel.State` and have the logic to determine whether it should be visible in there (so we keep the nice MVVM separation we currently have)